### PR TITLE
release the netcdf4 version

### PR DESCRIPTION
**Pull request by ocefpaf**
Nothing to see here, just debugging this...

### CODE REVIEW

File: pocean/utils.py [hunk 1]
```diff
@@ -284,6 +286,7 @@ def create_ncvar_from_series(ncd, var_name, dimensions, series, **kwargs):
                 raise ValueError('datatype error: boolean needs to be converted to string')
         except BaseException:
             # Fall back to a string type
+            kwargs.update({"zlib": False})
```
Comment (ocefpaf) [inline pocean/utils.py:289]: It turns out we cannot compress strings! This always failed but it was more forgiving in the past.

See https://github.com/Unidata/netcdf-c/pull/2231

File: pocean/utils.py [hunk 2]
```diff
@@ -264,6 +264,8 @@ def get_fill_value(var):
 
 def create_ncvar_from_series(ncd, var_name, dimensions, series, **kwargs):
     from pocean.cf import CFDataset
+    kwargs["zlib"] = kwargs.get("zlib", True)
+    kwargs["complevel"] = kwargs.get("complevel", True)
```
Comment (kwilcox) [inline pocean/utils.py:268]: This needs to be `1`

**Comment by kwilcox** [comment]
Glad this ended up being a clean fix!

**Review by kwilcox — CHANGES_REQUESTED** [review]

File: pocean/utils.py (hunk 2 above)
Comment (ocefpaf) [reply pocean/utils.py]: Ooops. Copy-n-pasta. Fixed in the next commit.

**Review by kwilcox — APPROVED** [review]

**Pull request merged**